wondrous-app/components/Common/CSVFileDropZone/styles.tsx

```diff
@@ -0,0 +1,60 @@
+import { Typography } from '@mui/material';
+import styled from 'styled-components';
+import DocumentIcon from 'components/Icons/DocumentIcon';
+import typography from 'theme/typography';
+import palette from 'theme/palette';
+
+export const DropZone = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 40px 0;
+  border: 2px dashed ${palette.grey78};
+  border-radius: 6px;
+  cursor: pointer;
+  color: ${palette.grey78};
+  font-size: 14px;
+  font-weight: 500;
+  transition: all 0.2s ease-out;
+
+  :hover {
```

Comment:
I think it should be `&:hover`, isn't it?

Reply:
both works actually, but do let me know in case you think using & explicitly is a better practice